File: KotlinLibrary/library/src/main/java/com/lee/library/widget/interadap/TextWatcherAdapter.java
package com.lee.library.widget.interadap;

import android.text.Editable;
import android.text.TextWatcher;

/**
 * @author jv.lee
 * @date 2020/4/24
 * @description
 */
public abstract class TextWatcherAdapter implements TextWatcher {
    @Override
    public void afterTextChanged(Editable s) {
    }

    @Override
    public void beforeTextChanged(CharSequence s, int start, int count, int after) {
    }

    @Override
    public void onTextChanged(CharSequence s, int start, int before, int count) {
    }

}

File: KotlinLibraryX/library/src/main/java/com/lee/library/interadp/TextWatcherAdapter.kt
package com.lee.library.interadp

import android.text.Editable
import android.text.TextWatcher

/**
 * 文本输入接口实现帮助类
 * @author jv.lee
 * @date 2021/11/26
 */
interface TextWatcherAdapter : TextWatcher {
    override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {

    }

    override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {

    }

    override fun afterTextChanged(s: Editable?) {

    }
}